cupy/math/window.py

```diff
@@ -2,10 +2,13 @@
 
 import cupy
 from cupy import core
-from cupy.creation import basic
-from cupy.creation import from_data
-from cupy.creation import ranges
-from cupy.math import trigonometric
+
+_blackman_kernel = core.ElementwiseKernel(
+    "float32 alpha",
+    "float64 out",
+    """
+    out = 0.42 - (0.5 * cos(i * alpha)) + (0.08 * cos(2 * alpha * i));
```

Comment:
Would you remove extra parentheses?
```suggestion
    out = 0.42 - 0.5 * cos(i * alpha) + 0.08 * cos(2 * alpha * i);
```